rresult.opam

```diff
@@ -1,19 +1,12 @@
 opam-version: "2.0"
 name: "rresult"
-synopsis: """Result value combinators for OCaml"""
 maintainer: ["Daniel Bünzli <daniel.buenzl i@erratique.ch>"]
 authors: ["The rresult programmers"]
-homepage: "https://erratique.ch/software/rresult"
-doc: "https://erratique.ch/software/rresult/doc/Rresult"
-dev-repo: "git+https://erratique.ch/repos/rresult.git"
+homepage: "https://github.com/dune-universe/rresult"
 bug-reports: "https://github.com/dbuenzli/rresult/issues"
 license: ["ISC"]
 tags: ["result" "error" "org:erratique"]
-depends: ["ocaml" {>= "4.08.0"}
-          "ocamlfind" {build}
-          "ocamlbuild" {build}
-          "topkg" {build & >= "1.0.3"}]
-build: [["ocaml" "pkg/pkg.ml" "build" "--dev-pkg" "%{dev}%"]]
+synopsis: """Result value combinators for OCaml"""
```

Comment:
Is there any particular reason why you're reordering the fields? I could imagine that merging that in the future might lead to more merge conflicts, but it is well possible that this has some advantage.

Reply:
No, it was to follow the same construction as other monorepo. I'll put it back to avoid future conflicts.